packages/driver/src/cypress/error_messages.js

```diff
@@ -814,7 +814,7 @@ module.exports = {
       docsUrl: 'https://on.cypress.io/api',
     },
     invalid_overwrite: {
-      message: 'Cannot overwite command for: `{{name}}`. An existing command does not exist by that name.',
+      message: 'Cannot overwrite command for: `{{name}}`. An existing command does not exist by that name.',
```

Comment:
A little extra housekeeping. 🧹 This is a separate commit in case I need to remove to keep this PR more focused!